include/deal.II/grid/reference_cell.h

```diff
@@ -338,6 +338,27 @@ ReferenceCell::operator!=(const ReferenceCell &type) const
 
 
 
+namespace internal
+{
+  namespace ReferenceCell
+  {
+    DEAL_II_CONSTEXPR dealii::ReferenceCell
```

Comment:
```suggestion
    inline DEAL_II_CONSTEXPR dealii::ReferenceCell
```

Reply:
Wow. You pushed the suggestion the same time I clicked "Comment". :smile:

Reply:
Yup, discovered that a couple of minutes ago as well. Let's see whether this iteration works -- I'm glad I broke this patch out of the bigger one, it turns out to be unexpectedly difficult...

Reply:
We're on it in this project :-)